local/gugcat/amd/src/main.js

```diff
@@ -112,7 +112,19 @@ define(['jquery', 'core/str' ], function($, Str) {
                 $(".togglemultigrd").show();
                 break;
             case btn_release:
-                document.getElementById('release-submit').click();
+                (async () => {
+                    var body_content = await Str.get_string('modalreleaseprovisionalgrade', 'local_gugcat');
+                    var str_cancel =  await Str.get_string('cancelrelaseprovisionalgrade', 'local_gugcat');
```

Comment:
Just change this to Str.get_strings(keys).then(function(langStrings), so that it will just call the server to get the 3 strings once,  rather than one call per string.